cluster-sync/external/provider.sh

```diff
@@ -21,7 +21,7 @@ function up() {
 
 function configure_storage() {
   if [[ $KUBEVIRT_STORAGE == "hpp" ]] ; then
-    _kubectl apply -f ./cluster-sync/external/resources/machineconfig-worker.yaml
+    _kubectl apply -f https://github.com/kubevirt/hostpath-provisioner-operator/blob/master/contrib/machineconfig-selinux-hpp.yaml
```

Comment:
Maybe we should check machinconfig crd installed?  SOmething similar to:

https://github.com/kubevirt/containerized-data-importer/blob/master/cluster-sync/ephemeral_provider.sh#L56-L58